gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -50,28 +64,29 @@ resources:
      cloud_config:
        manage_etc_hosts: true
        users:
-           - name: syseleven
-             gecos: SysEleven Stack user
+           - name: { get_param: username }
+             gecos: [ { get_param: username }, "Stack user"]
              sudo: ALL=(ALL) NOPASSWD:ALL
              shell: /bin/bash
        write_files:
          -  content: |
               #!/usr/bin/env bash
+              USERNAME=$1
+              echo "# Started SSH key injection and motd setup"
               # wait for valid network
               until ping -c 1 syseleven.de > /dev/null; do sleep 2; done
               # wait for a valid user
-              until id syseleven > /dev/null; do sleep 2; done
-              mkdir /home/syseleven/.ssh
-              curl -s http://169.254.169.254/latest/meta-data/public-keys/0/openssh-key > /home/syseleven/.ssh/authorized_keys
-              cat <<EOF> /home/syseleven/.ssh/config
-              Host 185.56.* 
-               UserKnownHostsFile=/dev/null
-               StrictHostKeyChecking=no
-              EOF
-              chown -R syseleven: /home/syseleven
+              until id ${USERNAME} > /dev/null; do sleep 2; done
+              echo "Found id: ${USERNAME}"
+              mkdir /home/${USERNAME}/.ssh
+              # create empty file for next command not to run into an error
+              touch /home/${USERNAME}/.ssh/authorized_keys
+              curl -s http://169.254.169.254/latest/meta-data/public-keys/0/openssh-key >> /home/${USERNAME}/.ssh/authorized_keys
```

Comment:
I think parametrizing the username does not add enough value to justify the added complexity.